src/styles/_catalog.scss

```
@@ -1,22 +1,36 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: 200px;
+  gap: 48px;
 
-  max-width: 944px;
   margin: 0 auto;
 
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: 200px 200px;
+    }
+  }
```

You don't need to use selector when media query is nested.
```suggestion
  @media (min-width: 488px) {
    grid-template-columns: 200px 200px;
  }
```